docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,107 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1 with Flexible Launch Control(FLC)
+
+## Install Git and Clone the Open Enclave SDK repo
+
+Download and install Git for Windows from [here](https://git-scm.com/download/win)
+
+Clone the Open Enclave SDK
+
+```powershell
+git clone https://github.com/openenclave/openenclave.gits
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project prerequisites
+
+First, change directory into the openenclave repository:
+
+```powershell
+cd openenclave
+```
+
+Run the following from powershell to deploy all the prerequisites for building Open Enclave(including Intel's DCAP primitives and Azure's DCAP library).
+
+```scripts/install-windows-prereqs.ps1```
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -LaunchConfiguration SGX1FLC -DCAPClientType Azure
```

Comment:
Why does this require DAPClientType Azure?  In contrast, WindowsManualSGX1FLCDCAPPrereqs.md says it is not, since it has both Intel and Azure.   Why the inconsistency?

Reply:
@dthaler 
Please note that this is existing documentation which I am just splitting
https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/GettingStarted.Windows.md

Currently, to do attestation in Azure, we need the Azure DCAP client. We dont have an alternate DCAP client implementation from either Intel or a third party.

In the manual install docs, I have the following
## [Azure DCAP client for Windows](https://github.com/Microsoft/Azure-DCAP-Client/tree/master/src/Windows) [optional]

Note that this is optional since you can choose an alternate implementation of the DCAP client or create your own.

Please let me know if this clears this up.

Reply:
It doesn't clear anything up in this file.  As I noted above, in the other file in your change you have both Azure and intel DCAP clients, and this note "The Azure DCAP client for Windows is necessary if you would like to perform enclave attestation on a Azure Confidential Computing VM."   This makes it clear that the Azure DCAP client is only relevant for ACC uses and not for Edge uses.  This page on the other hand, has no such statement, and as such is wrong, since the Azure DCAP client won't work on the edge.

Reply:
@dthaler I have documented the various cases now and stated that Azure DCAP client is required for attestation on ACC.

Reply:
also opened issue #2177 to try out Intel's DCAP client and change documentation and scripts once it is available.